src/clj/rems/css/styles.clj

```diff
@@ -262,12 +262,8 @@
    :margin-bottom (u/px 16)})
 
 (defn- generate-solid-form-group []
-  {:position "relative"
-   :border "2px solid #eee"
-   :margin 0
-   :padding (u/rem 1)
-   :padding-right (u/rem 2)
-   :border-radius (u/rem 0.4)})
+  (assoc (generate-dashed-form-group)
+         :border "2px solid #eee"))
```

Comment:
I think it could be slightly more intuitive if this was written with a `(generate-group)` common helper that both variants can use (solid, dashed). However this is fine too.

Reply:
Why not!